.circleci/config.yml

```diff
@@ -72,3 +90,19 @@ jobs:
             aws s3 cp build/asset-manifest.json s3://dataworld-chartbuilder-us-east-1/asset-manifest.json --cache-control max-age=300
           environment:
             AWS_PROFILE: artifacts
+workflows:
+  version: 2
```

Comment:
With the config version `2.1`, the version field should be dropped from the `workflows` section or "bad things will happen" - https://discuss.circleci.com/t/circleci-2-1-config-overview/26057

Reply:
oh wow haha